src/components/implementation/tests/micro_booter/mb_tests.c

```diff
@@ -408,85 +289,158 @@ spinner_cyc(void *d)
 {
 	cycles_t *p = (cycles_t *)d;
 
-	while (1) rdtscll(*p);
+	while (1)
+        rdtscll(*p);
+}
+
+static void
+test_2timers(void)
+{
+	if (EXPECT_LL_NEQ(0, exec_cluster_alloc(&bt[cos_cpuid()].p, parent, &bt[cos_cpuid()].p, BOOT_CAPTBL_SELF_INITRCV_CPU_BASE), "2 timer test"))
+		return;
+	if (EXPECT_LL_NEQ(0, exec_cluster_alloc(&bt[cos_cpuid()].c, spinner, &bt[cos_cpuid()].c, bt[cos_cpuid()].p.rc), "2 timer test"))
```

Comment:
Please read the composite style guide and fix your style throughout the changes.